cnf-certification-test/identifiers/identifiers.go

```diff
@@ -938,5 +942,14 @@ that there are no changes to the following directories:
 		Description:           formDescription(TestPodRequestsAndLimitsIdentifier, `Check that containers have resource requests and limits specified in their spec.`),
 		Remediation:           RequestsAndLimitsRemediation,
 		BestPracticeReference: bestPracticeDocV1dot4URL + " Section 4.6.11",
+		ExceptionProcess:      NoDocumentedProcess,
+	},
+	TestPersistentVolumeReclaimPolicyIdentifier: {
+		Identifier:            TestPersistentVolumeReclaimPolicyIdentifier,
+		Type:                  informativeResult,
+		Description:           formDescription(TestPersistentVolumeReclaimPolicyIdentifier, `Check that containers have resource requests and limits specified in their spec.`),
```

Comment:
This description seems to be a copy/paste from `TestPodRequestsAndLimitsIdentifier`.

Reply:
Yes, good catch.  Fixed.